src/exponential.jl

```diff
@@ -121,6 +121,12 @@ function quadratic_expansion(A::IntervalMatrix, α::Real, β::Real)
     return B
 end
 
+function quadratic_expansion(x::Interval, α::Real, β::Real)
```

Comment:
i suggest to add a basic docstring:
 
````
"""
    quadratic_expansion(x::Interval, α::Real, β::Real)

Compute `(α + βx) x` with a single-use expression.

### Input

...

### Output

...
"""
function quadratic_expansion(x::Interval, α::Real, β::Real)
````
```suggestion
function quadratic_expansion(x::Interval, α::Real, β::Real)
```

Reply:
oh, i take back my comment, given that the docstring for the function just above this one is OK.

Reply:
does "single-use" here mean "rewritten so that it doesn't suffer from dependency problem"?

Reply:
Yes.

Reply:
yes. i read it abbreviated SUE somewhere